src/modalities/trainer.py

```diff
@@ -54,6 +80,26 @@ def _train_batch(
         loss_fun: Loss,
         micro_batch_id: int,
     ) -> Tuple[bool, int, torch.Tensor, Optional[torch.Tensor], Optional[torch.Tensor]]:
```

Comment:
```suggestion
    ) -> Tuple[bool, int, torch.Tensor, Optional[torch.Tensor]]:
```
Only 4 variables are returned.